package.json

```
@@ -0,0 +1,33 @@
+{
+  "name": "tiny-lobster",
+  "version": "0.1.0",
+  "description": "My teensy-tinsy node.js pet",
+  "main": "index.js",
+  "repository": "git@github.com:octocrux/tiny-lobster.git",
+  "author": "octocrux <cate.petrusevich@gmail.com>",
+  "license": "MIT",
+  "private": true,
+  "scripts": {
+    "clean": "rm -rf dist/* && mkdir -p dist",
```

This line is not cross-platoform. Use specific tools(rimraf and make-dir-cli) for that.
```suggestion
    "clean": "rimraf dist/* && make-dir dist",
```